packages/core/upload/server/controllers/utils/find-entity-and-check-permissions.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const _ = require('lodash');
+const { contentTypes: contentTypesUtils } = require('@strapi/utils');
+const { NotFoundError, ForbiddenError } = require('@strapi/utils').errors;
```

Comment:
Should these two be combined? Wondering why the linter doesn't complain about it 🤔

Reply:
We have a different linter for backend and frontend :p
Does it have a performance impact? Otherwise I find it more readable to do this way. And I would even do that :
```js
const { CREATED_BY_ATTRIBUTE } = require('@strapi/utils').contentTypes.constants;
const { NotFoundError, ForbiddenError } = require('@strapi/utils').errors;
```

Reply:
I don't think it does has a serious performance impact tbh. I was confused, because in the frontend the linter would complain about that ...